src/main/java/org/benetech/servicenet/service/SharedLogicService.java

```diff
@@ -0,0 +1,20 @@
+package org.benetech.servicenet.service;
+
+import org.benetech.servicenet.domain.Location;
+import org.benetech.servicenet.domain.OpeningHours;
+import org.benetech.servicenet.domain.Phone;
+import org.benetech.servicenet.domain.RegularSchedule;
+import org.benetech.servicenet.domain.Service;
+
+import java.util.Set;
+
+public interface SharedLogicService {
```

Comment:
Maybe SharedImportService? As it's in the package where import-related classes are not the only services?